src/styles/calendar.scss

```diff
@@ -0,0 +1,62 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.sections {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $my-var; // Ширина календаря
+  gap: $width-gap; // Проміжки між днями
+
+  &__day {
+    width: $width-boxs; // Ширина дня
+    height: $width-boxs; // Висота дня
```

Comment:
The variables `$width-boxs` should be named in singular form and with a more descriptive name, such as `$day-box-size`, to better represent the element it styles.